verification/curator-service/api/test/users.test.ts

```diff
@@ -23,37 +25,45 @@ afterAll(() => {
 
 beforeEach(async () => {
     await User.deleteMany({});
+    await Session.deleteMany({});
+});
+
+afterEach(async () => {
```

Comment:
Can be after all tests for a final cleanup since we delete them all in beforeEach